CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ Here's a quick guide to create a pull request for your WordPress-Android patch:
 
 4. Setup your build environment (see [build instructions in our README][build-instructions]) and start hacking the project. You must follow our [code style guidelines][style], write good commit messages, comment your code and write automated tests.
 
-5. When your patch is ready, [submit a pull request][pr]. Add some comments or screen shots to help us.
+5. When your patch is ready, [submit a pull request][pr]. Make sure the pull request addresses one of our existing issues and its description refers to that issue. Add some comments or screen shots to help us.
```

Comment:
In some cases someone may submit a PR that doesn't reference an existing issue – do we want to make them create one?

I'd say we can just add a note that says something like "Please be aware that we'll only merge substantive PRs – any that don't clearly improve the codebase will be closed without merging". IMHO that makes our intent clearer? 

WDYT?